### ISSUE TITLE
Error installing librosa

### ISSUE THREAD

**Issue by MartinDelille** [librosa/librosa#533]
I have the following error when trying to install *librosa* with *pip*:

```
$ pip install librosa
Collecting librosa
  Using cached librosa-0.5.0.tar.gz
Collecting audioread>=2.0.0 (from librosa)
  Using cached audioread-2.1.4.tar.gz
Collecting numpy>=1.8.0 (from librosa)
  Using cached numpy-1.12.1-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting scipy>=0.13.0 (from librosa)
  Using cached scipy-0.19.0-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting scikit-learn>=0.14.0 (from librosa)
  Using cached scikit_learn-0.18.1-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting joblib>=0.7.0 (from librosa)
  Using cached joblib-0.11-py2.py3-none-any.whl
Collecting decorator>=3.0.0 (from librosa)
  Using cached decorator-4.0.11-py2.py3-none-any.whl
Requirement already satisfied: six>=1.3 in /usr/local/lib/python2.7/site-packages (from librosa)
Collecting resampy>=0.1.2 (from librosa)
  Using cached resampy-0.1.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/dd/626hkm4d5356jx9xp8__4zxc0000gn/T/pip-build-xGNoer/resampy/setup.py", line 6, in <module>
        import numpy as np
      File "/usr/local/lib/python2.7/site-packages/numpy/__init__.py", line 142, in <module>
      File "/usr/local/lib/python2.7/site-packages/numpy/add_newdocs.py", line 13, in <module>
      File "/usr/local/lib/python2.7/site-packages/numpy/lib/__init__.py", line 8, in <module>
      File "/usr/local/lib/python2.7/site-packages/numpy/lib/type_check.py", line 11, in <module>
      File "/usr/local/lib/python2.7/site-packages/numpy/core/__init__.py", line 14, in <module>
    ImportError: cannot import name multiarray

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/dd/626hkm4d5356jx9xp8__4zxc0000gn/T/pip-build-xGNoer/resampy/
```

I'm using:
- *OSX 10.11.6*
- *Python 2.7.13*
- *pip 9.0.1*

**Comment by bmcfee** [comment]
This is a noted issue for resampy: https://github.com/bmcfee/resampy/issues/34  and is not specific to librosa.

TLDR: the quick fix is to install numpy first and then it should work.

The less quick, but long-term better fix (IMO) is to use conda packaging instead of pip, if that's a possibility for you.

**Comment by inspiralpatterns** [comment]
@bmcfee that worked for me, same specifics. 
Thanks a lot!

**Comment by bmcfee** [comment]
This issue should be resolved by resampy 0.2.0.  If it's still not working, the install issue will be due to some other problem unrelated to the above, so I'm closing this out.
